src/safe-upgrade-tool/run.md

```diff
@@ -0,0 +1,137 @@
+---
+group: software-update-guide
+title: Run the tool
+functional_areas:
+  - Upgrade
+---
+
+The Magento Safe Upgrade Tool (SUT) is a command line (CLI) tool that checks a Magento instance against a specific version by analyzing all the non-Magento modules installed on it.
+
+potential problems that must be fixed in your custom code before attempting to upgrade to a newer version of Magento.
+
+The tool returns a list of errors and warnings that you must address before upgrading to a new version of Magento.
+
+## Use SUT
+
+### Memory limitations
+
+The SUT uses memory on your Magento instance to run—even if you installed the SUT on a different machine. If you are running a system with large third-party modules and files, the SUT requires a high amount of RAM.
+
+### Executing the tool
+
+Execute the tool by running the following command:
+
+```bash
+bin/sut upgrade:check INSTALLATION_DIR
+```
+
+{:.bs-callout-info}
+We recommend running `php -d memory_limit=-1 /bin/sut` to avoid memory limitations.
+
+The `INSTALLATION_DIR` value is the directory where your Magento instance is located.
+
+To see SUT command options and help:
+
+```bash
+bin/sut --help
+```
+
+### GraphQL schema compatibility verification
+
+The SUT also provides the option to introspect two GraphQL endpoints and compare their schemas looking for breaking and dangerous changes between them:
```

Comment:
It is important to mention, that in this case they must have **running** "instance before" and "instance after" the upgrade.